constants.py

```diff
@@ -66,12 +71,24 @@ class DisplayEndingType(StrEnum):
 RAW_DESCRIPTOR_SET = 128
 """The ID of the raw data packet that the IMU sends"""
 
+# This is used by all queues to keep things consistent:
+MAX_FETCHED_PACKETS = 15
+"""This is used to limit how many packets we fetch from the imu at once."""
+
+# Timeouts for get() queue operations:
+MAX_GET_TIMEOUT_SECONDS = 100  # seconds
+"""The maximum amount of time in seconds to wait for a get operation on the queue."""
+
+# Max bytes to put/get from the queue at once:
+MAX_SIZE_BYTES = 1000 * 1000 * 20  # 20 Mb
```

Comment:
rename to MAX_NUMBER_OF_BYTES or something. size makes me think byte size

Reply:
that's the exact name the library used. How about `BUFFER_SIZE_IN_BYTES`?

Reply:
yeah thats fine